plugins/catalog/src/components/EntityPage/EntityPage.tsx

```diff
@@ -184,42 +238,7 @@ export const EntityPage: FC<{}> = () => {
 
       {entity && (
         <>
-          <HeaderTabs tabs={tabs} />
-
-          <Content>
-            <Grid container spacing={3}>
-              <Grid item sm={4}>
-                <EntityMetadataCard entity={entity} />
-              </Grid>
-              {entity.metadata?.annotations?.[
-                'backstage.io/jenkins-github-folder'
-              ] && (
-                <Grid item sm={4}>
-                  <JenkinsLastBuildWidget entity={entity} branch="master" />
-                </Grid>
-              )}
-              {entity.metadata?.annotations?.[
-                'backstage.io/jenkins-github-folder'
-              ] && (
-                <Grid item sm={8}>
-                  <JenkinsBuildsWidget entity={entity} />
-                </Grid>
-              )}
-              {entity.metadata?.annotations?.[
-                'backstage.io/github-actions-id'
-              ] && (
-                <Grid item sm={3}>
-                  <GithubActionsWidget entity={entity} branch="master" />
-                </Grid>
-              )}
-            </Grid>
-            <Grid item sm={8}>
-              <SentryIssuesWidget
-                sentryProjectId="sample-sentry-project-id"
-                statsFor="24h"
-              />
-            </Grid>
-          </Content>
+          <TabNavigator<{ entity: Entity }> tabs={tabs} tabData={{ entity }} />
```

Comment:
i'm not sure about this data passing - how about

```
  const tabs = [
    {
      id: 'overview',
      label: 'Overview',
      content: () => <EntityOverviewPage entity={entity} />
    }
```

That way we get to break out all of those page contents elsewhere and don't have to invent a custom passing mechanism. And then we could just render the `HeaderTabs` directly in here and handle its onChange with navigation in the same place as the other knowledge in here